.gitignore

```diff
@@ -6,3 +6,4 @@
 erl_crash.dump
 *.ez
 .env*
+/.elixir_ls
```

Comment:
I think a good practice is to add IDE stuff to global ignore.

Reply:
Well it's depend whether people familiar with this or not. For example `.vs_code` folder is very popular in many open source projects.

Similarly `.elixir_ls` quite popular also, from my experience

Reply:
I think we should ignore the only project related stuff to keep all consistent. Otherwise, ignore all IDE internal files and project